terraform/instances.tf

```diff
@@ -615,35 +615,12 @@ resource "aws_instance" "worker" {
 ////////////////////////////////
 // Template Files
 
-data "template_file" "hab_sup" {
+data "template_file" "sup_service" {
   template = "${file("${path.module}/templates/hab-sup.service")}"
 
   vars {
-    flags               = "--auto-update --channel ${var.release_channel} --events hab-eventsrv.default --listen-gossip 0.0.0.0:${var.gossip_listen_port} --listen-http 0.0.0.0:${var.http_listen_port}"
-    gossip_listen_port  = "${var.gossip_listen_port}"
-    peer_ip             = "${aws_instance.datastore.0.private_ip}"
-    log_level           = "${var.log_level}"
-  }
-}
-
-data "template_file" "hab_sup_permanent" {
-  template = "${file("${path.module}/templates/hab-sup.service")}"
-
-  vars {
-    flags               = "--auto-update --channel ${var.release_channel} --events hab-eventsrv.default --listen-gossip 0.0.0.0:${var.gossip_listen_port} --listen-http 0.0.0.0:${var.http_listen_port} --permanent-peer"
-    gossip_listen_port  = "${var.gossip_listen_port}"
-    peer_ip             = "${aws_instance.datastore.0.private_ip}"
-    log_level           = "${var.log_level}"
-  }
-}
-
-data "template_file" "hab_sup_seed" {
-  template = "${file("${path.module}/templates/hab-sup.service")}"
-
-  vars {
-    flags               = "--auto-update --channel ${var.release_channel} --events hab-eventsrv.default --listen-gossip 0.0.0.0:${var.gossip_listen_port} --listen-http 0.0.0.0:${var.http_listen_port} --permanent-peer"
-    gossip_listen_port  = "${var.gossip_listen_port}"
-    peer_ip             = "127.0.0.1"
-    log_level           = "${var.log_level}"
+    flags     = "--auto-update --channel ${var.release_channel} --events hab-eventsrv.default --listen-gossip 0.0.0.0:${var.gossip_listen_port} --listen-http 0.0.0.0:${var.http_listen_port}"
+    peers     = "${var.peers}"
```

Comment:
Oh there's no difference, I could just totally pass it as a flag above. This is just an artifact from the previous form that this resource took. I'll simplify it :)

The variable is required so it can't be empty